### DISCUSSION TITLE
Which release versions work

### DISCUSSION THREAD

**Discussion by sohanshow** [thatmarcel/beepserv-rewrite#7] [Q&A] [answered]
Device I tried on: Iphone 6s (iOS 15.8.1) with rootless jailbreak.

a686640->
Works! (iOS 15.8.1)

e90552f->
- Stuck on No valid cached validation data exists
- Requesting new validation data
- Received request for validation data... (stuck here)

d0bfb0a -> 
- Stuck on No valid cached validation data exists
- Requesting new validation data
- Received request for validation data... (stuck here)

8033603 -> 
- Stuck on No valid cached validation data exists
- Requesting new validation data
- Received request for validation data... (stuck here)

e0477ea -> Not Tested.

a19b252 -> Works!

A ton of thanks to Marcel to make this project possible. Great work.

**Comment by thatmarcel** [comment]
Thanks a lot for the detailed overview of which versions work on the device and what the last logs are! :)

Do you recall what the logs said about offsets on the newest 3 versions? Does it say "Finding offsets failed" or "Found offsets" or does it say nothing at all (these logs should start with `[IdentityServices]`)? Also, which iOS version are you running? I'm trying to figure out where the newest versions get stuck, specifically on the 6s.

**Reply by thatmarcel** [reply]
The newest release ([a686640](https://github.com/thatmarcel/beepserv-rewrite/releases/tag/a686640)) should fix the fallback not working on the 6s, feel free to try it and let me know how it goes.

**Reply by sohanshow** [reply, marked as answer]
Yes. This one works. Thanks a ton for the work Marcel. You should lowkey contact Beeper. I already mentioned about your work in support. Appreciate the great work friend.
